src/main/java/uk/gov/hmcts/ccd/data/casedetails/DefaultCaseDetailsRepository.java

```diff
@@ -185,7 +185,7 @@ public PaginatedSearchMetadata getPaginatedSearchMetadata(MetaData metaData, Map
 
     private Query getQuery(MetaData metadata, Map<String, String> dataSearchParams, boolean isCountQuery) {
         Query query;
-        if (dataSearchParams.isEmpty()) {
+        if (dataSearchParams.isEmpty() && applicationParams.isJpaCriteriaSearchEnabled()) {
```

Comment:
cherry-picked from PR 529 until that is merged into master.

Reply:
`applicationParams.isJpaCriteriaSearchEnabled()` is temporary, can you create a ticket to remove it please? together with `getQueryByMetaData`

Reply:
sure, will create one.

Reply:
https://tools.hmcts.net/jira/browse/RDM-5700

Reply:
thanks. Added to the epic

Reply:
Perhaps we could add a comment in the code as well?